librz/include/sflib/freebsd-x86-32/sflib.h

```diff
@@ -0,0 +1,293 @@
+// SPDX-FileCopyrightText: 2004  Philippe Biondi <phil@secdev.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+/*
+ * sflib.h --- SFLib syscall library for FreeBSD/i386 
+ * see http://www.secdev.org/projects/shellforge.html for more informations
+ *
```

Comment:
I think for imported files better to set SPDX information in `.reuse/dep5` instead, without modifying originals.

Reply:
@XVilka Could you please let me know what should therefore be done in this case? I understand that those files should be intact because are imported, but I cannot see any similar reference of the SPDX header in the `.reuse/dep5` file. Once again, thanks in advance!